tox.ini

```diff
@@ -22,3 +22,8 @@ deps =
     black==18.9b0
 commands =
     black {posargs:--check setup.py faculty_cli test}
+
+[flake8]
+# Ignore long line errors, and rely on the formatting done properly by black
+# Also ignore "Line break occurred before a binary operator" also coming from black
+ignore = E501,W503
```

Comment:
Can we instead put this in `setup.cfg`, as in https://github.com/facultyai/faculty/blob/master/setup.cfg ?

It should then get picked up by linter plugins in peoples' editors.

Reply:
That's good info, thanks! Just getting to learn the number of different configs that one can use to set these tooling (just like `pyproject.toml` mentioned earlier). Will try that out too, and it makes complete sense putting it where it has the best effect.